docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '3.4'
+
+networks:
+  k6:
+  grafana:
+  timescaledb:
+
+services:
+  timescaledb:
+    image: timescale/timescaledb:latest-pg12
+    command: postgres -c shared_preload_libraries=timescaledb
+    networks:
+      - k6
+      - grafana
+      - timescaledb
+    ports:
+      - "5432:5432"
+    environment:
+      - PGDATA=/var/lib/postgresql/data/timescaledb
+      - POSTGRES_DB=k6
+      - POSTGRES_USER=k6
+      - POSTGRES_PASSWORD=k6
+
+  grafana:
+    image: grafana/grafana:latest
+    networks:
+      - grafana
+      - timescaledb
+    ports:
+      - "3000:3000"
+    environment:
+      - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin
+      - GF_AUTH_ANONYMOUS_ENABLED=true
+      - GF_AUTH_BASIC_ENABLED=false
+    volumes:
+      - ./grafana-datasource.yaml:/etc/grafana/provisioning/datasources/datasource.yaml
+
+  k6:
+    build: .
+    networks:
+      - k6
+    ports:
+      - "6565:6565"
+    environment:
+      - K6_OUT=timescaledb=postgresql://k6:k6@timescaledb:5432/k6
```

Comment:
I needed to provide username and password as otherwise, I was getting errors, and it might be a good idea to add them to the default settings ... somehow?

Reply:
What do you mean for **default settings**? I think we should add options for them, or not? :thinking:

Reply:
it's all configured through the url and I really prefer to stay that way as it's a postgres standard thing ;) ... I even wonder if we should move the pushInterval out of it and only have it as a ENV variable especially as I don't think it will be used all that much :thinking:

Reply:
if it makes sense - yes, but in this case it makes more sense as the URL configuration is basically baked in postgres so we just piggyback on it and our users can just use their knowledge of it.

PushInterval being the only specific one (and again IMO an unlikely one to be configured) can possibly only have ENV variable to be configured.

Reply:
cc @na-- @imiric @yorugac @inancgumus

Reply:
There will be similar situation in prometheus remote extension: I also added the docker-compose setup there.
In both cases, is docker-compose meant only for illustration purposes? If so, then we could leave these as is, in URL + maybe mention in readme. At least, until / unless it's merged into the core..